### PR TITLE
feat: add event based expiry column to index table

### PR DESCRIPTION
**Pull request by daywiss**
Signed-off-by: David <david@umaproject.org>

This adds a column on the index page to show event based expiry. Columns needed to be adjusted to fit better. It takes time to load event based data, so if the data isnt available it will show `-` until it loads. 

image

### CODE REVIEW

File: src/components/index/createRequestsTableCells.tsx [hunk 1]
```diff
@@ -97,6 +102,9 @@ function createRequestsTableCells(
         req.requestTx || req.proposeTx || req.disputeTx || req.settleTx
       }&chainId=${req.chainId}&oracleType=${req.oracleType}&eventIndex=${0}`;
 
+      const eventBased =
+        req.eventBased === false ? "No" : req.eventBased === true ? "Yes" : "-";
+
```
Comment (Tulun) [inline src/components/index/createRequestsTableCells.tsx:107]: That's the 3rd value to this? Is it sometimes undefined / null ?

**Review by Tulun — APPROVED** [review]
Small question otherwise LGTM

File: src/components/index/createRequestsTableCells.tsx (hunk 1 above)
Comment (smb2796) [reply src/components/index/createRequestsTableCells.tsx:107]: I'd guess this is for loading state

**Comment by smb2796** [comment]
Can we actually just change copy to `Event Based` on this page? Other than that, looks awesome!

**Comment by smb2796** [comment]
Oh also, can the ordering be changed to have `Event Based` column come second after request timestamp and before status?

**Pull request merged**